include/net/net_ip.h

```diff
@@ -586,6 +586,27 @@ static inline bool net_ipv4_addr_mask_cmp(struct net_if *iface,
 	return net_if_ipv4_addr_mask_cmp(iface, addr);
 }
 
+extern bool net_if_ipv4_addr_is_bcast(struct net_if *iface,
+				      const struct in_addr *addr);
+
+/**
+ * @brief Check if the given IPv4 address is a broadcast address.
+ *
+ * @param iface Interface to use. Must be a valid pointer to an interface.
+ * @param addr IPv4 address
+ *
+ * @return True if address is a broadcast address, false otherwise.
+ */
+static inline bool net_is_ipv4_addr_bcast(struct net_if *iface,
```

Comment:
s/net_is_ipv4_addr_bcast/net_ipv4_is_addr_bcast

Reply:
Lets not change this in this case, this function name is following the same format as the other `net_is_ipv4_*()` functions in that file.

Reply:
> his function name is following the same format as the other net_is_ipv4_*() functions in that file.

Would be good to rename them all then (in another PR).